index.js

```diff
@@ -14,7 +14,8 @@ ll = ll && ll.querySelector('hui-root');
 
 const screensaverConfig = {
   query: 'nature',
-  idle_time: 10,
+  idle_time: 5,
+  orientation: 'portrait',
```

Comment:
landscape should be the default, as in the README

Reply:
Yes, you are right.